### PR TITLE
#19886 Make Customize Toolbar be a property page

### LINKED ISSUE #19886: Add option to Customize Toolbar for SQL and Data editor preferences
**Is your feature request related to a problem? Please describe.**
For now we can customize SQL/Data Editor toolbars from the Main Preferences only 

**Describe the solution you'd like**
It will be great if Customize toolbars options will be available from "local" preferences for SQL and Data editors
image

### PR DESCRIPTION
**Pull request by E1izabeth**
Please, check that this preferences still can be changed, saved and restored to default.

image

image

### CODE REVIEW

**Review by ShadelessFox — APPROVED** [review]

**Review by LonwoLonwo — APPROVED** [review]

**Comment by MashaKorax** [comment]
verified

File: plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/preferences/PrefPageToolbarCustomization.java [hunk 1]
```diff
@@ -53,8 +53,10 @@
 import java.util.stream.Collectors;
 
 
-public class PrefPageToolbarCustomization extends AbstractPrefPage implements IWorkbenchPreferencePage {
-    
+public class PrefPageToolbarCustomization extends TargetPrefPage {
```
Comment (serge-rider) [inline plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/preferences/PrefPageToolbarCustomization.java]: Why TargetPrefPage? It is designed for connection-specific properties. AbstractPrefPage maybe better.

File: plugins/org.jkiss.dbeaver.core/plugin.xml [hunk 2]
```diff
@@ -1303,12 +1308,4 @@
                 group="%confirmation.group.general.actions"/>
     </extension>
 
-    <extension point="org.eclipse.ui.preferencePages">
```
Comment (serge-rider) [inline plugins/org.jkiss.dbeaver.core/plugin.xml]: So it will disappear from global preferences?

**Review by serge-rider — CHANGES_REQUESTED** [review]

File: plugins/org.jkiss.dbeaver.core/plugin.xml (hunk 2 above)
Comment (E1izabeth) [reply plugins/org.jkiss.dbeaver.core/plugin.xml]: No, it's still exists there

File: plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/preferences/PrefPageToolbarCustomization.java (hunk 1 above)
Comment (E1izabeth) [reply plugins/org.jkiss.dbeaver.core/src/org/jkiss/dbeaver/ui/preferences/PrefPageToolbarCustomization.java]: I used AbstractPrefPage before, it doesn't allow to bring this page to property settings

**Comment by MashaKorax** [comment]
verified

**Review by serge-rider — APPROVED** [review]

**Pull request merged**